crates/client/storage/src/overrides/mod.rs

```diff
@@ -76,6 +78,8 @@ pub trait StorageOverride<B: BlockT>: Send + Sync {
     fn nonce(&self, block_hash: B::Hash, address: ContractAddressWrapper) -> Option<NonceWrapper>;
     /// Returns the events for a provided block hash.
     fn events(&self, block_hash: B::Hash) -> Option<Vec<EventWrapper>>;
+    /// Returns the storage value for a provided key and block hash.
+    fn chain_id(&self, block_hash: B::Hash) -> Option<String>;
```

Comment:
Would make more sense to return an `Option<Felt252Wrapper>` here and then convert to string in the RPC.